### PR TITLE
[disaster dashboard] Main component of disaster dashboard

### PR DESCRIPTION
**Pull request by chejennifer**
- add main component that completes the disaster dashboard mvp

This is the last part of the changes for the disaster dashboard MVP here on dev:
https://dev.datacommons.org/dev/disaster-dashboard

### CODE REVIEW

File: server/routes/dev.py [hunk 1]
```diff
@@ -43,4 +45,7 @@ def screenshot(folder):
 def disaster_dashboard():
     if not os.environ.get('FLASK_ENV') in ['autopush', 'local', 'dev']:
         flask.abort(404)
-    return flask.render_template('dev/disaster_dashboard.html')
+    european_countries = json.dumps(
+        dc.get_places_in(["europe"], "Country").get("europe", []))
```
Comment (beets) [inline server/routes/dev.py:49]: thanks for adding this!

File: static/css/disaster_dashboard.scss [hunk 2]
```diff
@@ -15,3 +15,212 @@
...
+  margin-top: 1rem;
+}
+
+#disaster-dashboard-date-selector-input {
+  width: 12rem;
+  margin-left: 0.5rem;
+}
+
+.disaster-dashboard-map-section {
+  border: 1px solid rgba(0, 0, 0, 0.125);
```
Comment (beets) [inline static/css/disaster_dashboard.scss:73]: this could be a file variable (there are other opportunities as well but this is the most obvious one).

**Review by beets — APPROVED** [review]
thanks for also addressing comments from other pr's!

File: static/css/disaster_dashboard.scss (hunk 2 above)
Comment (chejennifer) [reply static/css/disaster_dashboard.scss]: done

**Pull request merged**